brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3026: Fix - Advance Heading line height issue in frontend with astra theme.

### Description

Subheading line-height is not same as backend in frontend when astra theme is activated.
https://d.pr/v/GjZ5Nj

### Screenshots


### Types of changes





### How has this been tested?


- Add a heading block on the new page.
- Add a subheading in the heading block
- Add some data in the subheading, and go to stylings.
- Make the subheading font-size to 80px-90px.
- Check front-end.

### Checklist:
